### DISCUSSION TITLE
Item animation on map

### DISCUSSION THREAD

**Discussion by ghost** [AscensionGameDev/Intersect-Engine#1478] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently when you kill an NPC and this one drops an item, you will see a static pic of it on the map/ground.

**Describe the solution you'd like**
Be able to choose an animation to show, instead of showing a static pic.

**Comment by BabyLoves** [comment]
You need search before open the issue/enhancement
https://www.ascensiongamedev.com/topic/5073-kash-shop-drop-animation/

**Comment by ghost** [comment]
> You need search before open the issue/enhancement
> https://www.ascensiongamedev.com/topic/5073-kash-shop-drop-animation/

Thanks, but that's not what I meant. That's a drop animation. I meant an item animation showing on the map instead of the current static pic.

**Comment by WeylonSantana** [comment]
Despite being an interesting idea, I think the implementation would be quite absurd, (I can't say because I'm still a beginner), what would the item be? animation from the animation editor? and who guarantees that the animation will follow the 32x32 standard in the frames, this could mess up the inventory even more if used incorrectly, and many would, so I'm against this
